.github/workflows/pylint.yml

```diff
@@ -0,0 +1,27 @@
+name: (Pylint + Flake8) linters + Black formatter
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Python 3.10
+        uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+          pip install pylint
+      - name: Running black formatter
+        uses: actions/checkout@v3
+      - uses: psf/black@stable
+      - name: Linting with pylint
+        run:  pylint $(git ls-files '*.py')
+      - name: Linting with flake8
+        uses: TrueBrain/actions-flake8@v2
```

Comment:
Powiedz mi która z tych linijek odpala formatter kodu black?

Reply:
23

Reply:
A czy kod nie byłby czytelniejszy, gdyby to było w linijce `22` akcja `Running black formatter` odpalałaby wtedy formatowanie kodu, a nie zajmowała się checkoutowniem?  

Logi też byłyby czytelniejsze   
https://github.com/KNR-PW/discord-bot/actions/runs/3665618503/jobs/6196809320   
![image](https://user-images.githubusercontent.com/45544416/206901857-2076a626-f7ea-4724-b249-ab88f7dda840.png)

Reply:
A poza tym już raz robiłeś checkouta w tym konfigu.

Reply:
Może rozwiązaniem pośrednim byłoby po prostu przeniesienie name:Running black formatter pod action checkout v3

Reply:
> Więc nie jestem pewien czy jest potrzeba robić go drugi raz. (tutaj akurat mogę się mylić, taki jest mój domysł)

Z tego co pamiętam, było niezbędne ponowne użycie checkouta w tej części kodu, jeżeli nie dla blacka to dla pylinta, dlatego zrobiłem tyle commitów, bo sprawdzałem różne wersje. Natomiast w oficjalnej dokumentacja używania blacka w github action, checkout pojawia się bezpośrednio przed wywołaniem blacka.

Reply:
Ale ja już to sam naprawię na szybko